### DISCUSSION TITLE
[gatsby-plugin-image] not displaying images on build

### DISCUSSION THREAD

**Discussion by JzeiJzei** [gatsbyjs/gatsby#30828] [Help]
## Description

I made a new project with the Gatsby default starter. After building the project the images do not display on Firefox 87.0 or on Google Chrome 89.0.4389.114. Also the links to different pages don't work.

### Steps to reproduce

- Install latest Gatsby default starter
- Run 'gatsby build'
- Open index.html from public folder with Firefox/Chrome

### Expected result

The gatsby-astronaut.png should be visible on the page along with the favicon.
Links to other pages should work as well.

### Actual result

Images are not displayed.
Links do not work.

### Environment

  System:
    OS: Windows 7 6.1.7601
    CPU: (4) x64 Intel(R) Core(TM) i5 CPU         750  @ 2.67GHz
  Binaries:
    Node: 12.18.3 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.6 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 89.0.4389.114
  npmPackages:
    gatsby: ^3.2.1 => 3.2.1 
    gatsby-plugin-gatsby-cloud: ^2.2.0 => 2.2.0 
    gatsby-plugin-image: ^1.2.0 => 1.2.0 
    gatsby-plugin-manifest: ^3.2.0 => 3.2.0 
    gatsby-plugin-offline: ^4.2.0 => 4.2.0 
    gatsby-plugin-react-helmet: ^4.2.0 => 4.2.0 
    gatsby-plugin-sharp: ^3.2.0 => 3.2.0 
    gatsby-source-filesystem: ^3.2.0 => 3.2.0 
    gatsby-transformer-sharp: ^3.2.0 => 3.2.0 
  npmGlobalPackages:
    gatsby-cli: 3.1.0

Any help would be appreciated, as I can't really tell what's causing this to occur.

**Comment by LekoArts** [comment]
The build artifacts need to be served by a webserver, you can't just open the `index.html` file with a browser. You can use `gatsby serve` to show the page at `localhost:9000`
